homeassistant/components/ness_alarm/alarm_control_panel.py

```diff
@@ -56,10 +58,10 @@ class NessAlarmPanel(AlarmControlPanelEntity):
         | AlarmControlPanelEntityFeature.TRIGGER
     )
 
-    def __init__(self, client: Client, name: str) -> None:
+    def __init__(self, client: Client, panel_name) -> None:
         """Initialize the alarm panel."""
         self._client = client
-        self._attr_name = name
+        self._attr_name = panel_name if panel_name else "Alarm Panel"
```

Comment:
In what scenarios is `panel_name` `None`/undefined? I believe that the default should be set when config is validated.